functions-methods.js

```diff
@@ -33,4 +64,29 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+const email5 = "n.eeken@novi.nl"
+const email6 = "tessmellink@novi.nl"
+const email7 = "n.eekenanovi.nl"
+const email8 = "n.eeken@novinl."
+const email9 = "tessmellink@novi,nl"
+
+function checkEmailValidity(email){
+    let boolean = true
+    if (email.includes("@") && !email.includes(",") && email.substring(email.length -1) !== "."){
+//  is email.substring(email.length -1) !== "." correct?
+
+        boolean = true
+    }
+    else { boolean = false
+    }
+
+    return boolean
```

Comment:
Je kunt de uitkomst van het statement meteen teruggeven:
```suggestion
    return email.includes("@") && !email.includes(",") && email.substring(email.length -1) !== ".";
```